docs/README.kr.md

```diff
@@ -11,17 +11,33 @@
 <br></br>
 [![Deploy to now](https://deploy.now.sh/static/button.svg)](https://deploy.now.sh/?repo=https://github.com/dawnlabs/carbon&env=TWITTER_CONSUMER_KEY&env=TWITTER_CONSUMER_SECRET&env=TWITTER_ACCESS_TOKEN_KEY&env=TWITTER_ACCESS_TOKEN_SECRET&env=LOGS_SECRET_PREFIX)
 
+
+##### Translations
```

Comment:
I just meant can you remove these line: https://github.com/dawnlabs/carbon/pull/763/files/42131c69d2c87a876def06d96be53965295df3bc#diff-8c0a439f8e009dcb2fdd6b1aca936db9R15

Lines 15 though 29. There should be no "##### Translations" section in this file

Reply:
@mfix22 yes. i miss understand words. thanks :)